data/json/items/book/maps.json

```
@@ -77,6 +77,21 @@
       "message": "You add roads and points of interest to your map."
     }
   },
+  {
+    "id": "subwaymap",
+    "copy-from": "abstractmap",
+    "type": "GENERIC",
+    "name": "subway maintenance map",
+    "looks_like": "roadmap",
+    "description": "This is a map of subway tunnels formerly used by public maintenance workers.  Using it will add subway lines and underground stations to your map.",
+    "color": "light_blue",
+    "use_action": {
+      "type": "reveal_map",
+      "radius": 180,
+      "terrain": [ "subway", "sub_station" ],
```

You can use different overmap terrain match type if you need it, e.g.:

```suggestion
"terrain": [  { "om_terrain": "subway", "om_terrain_match_type": "TYPE" }, "sub_station" ],
```

---

Thanks—I did something like this in previous revisions, but was happier with the “contains” matching I got from the plain string.